hrms/hr/doctype/interview/interview.py

```diff
@@ -26,6 +26,7 @@ def on_submit(self):
 			frappe.throw(
 				_("Only Interviews with Cleared or Rejected status can be submitted."), title=_("Not Allowed")
 			)
+		self.update_job_applicant_status()
```

Comment:
There could be multiple rounds of interviews for the same applicant. In that case, we cannot silently mark the applicant as Accepted.

Currently, the applicant is automatically marked as Accepted when a Job Offer is created for them.

Reply:
We could probably add a `msgprint` with a `server_action` on submit to allow the user to decide whether to update the status or not.

https://frappeframework.com/docs/user/en/api/py-dialog#frappe-msgprint

Something like this:
```python
frappe.msgprint(
	message,
	title=_("Update Job Applicant Status"),
	primary_action={
		"label": _("Mark as {0}").format(status),
		"server_action": "hrms.hr.doctype.interview.interview.update_job_applicant_status",
		"args": {
			"status": status,
		},
	},
)
```

Reply:
Yes, this way, users will also have control over updating the job applicant status, even if there are multiple interview rounds. I will make the necessary changes and update the PR.

Reply:
Did the changes as discussed

**Output:** 

### 1. 
<img width="1469" alt="Screenshot 2023-09-28 at 5 04 02 PM" src="https://github.com/frappe/hrms/assets/69882648/15ef3fb1-1e62-4484-8a31-b8c40234e3e0">

### 2. 
<img width="567" alt="Screenshot 2023-09-28 at 5 04 09 PM" src="https://github.com/frappe/hrms/assets/69882648/330c4698-6446-424b-aa61-9043bfdf1c3f">

### 3. 
<img width="1332" alt="Screenshot 2023-09-28 at 5 04 40 PM" src="https://github.com/frappe/hrms/assets/69882648/a414e4bf-6d70-40d3-bd7c-b3a1a02d09b1">